app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -225,19 +239,30 @@ public Study createSynapseProjectTeam(Long userId, Study study) throws SynapseEx
 
     public Study updateStudy(Study study, boolean isAdminUpdate) {
         checkNotNull(study, Validate.CANNOT_BE_NULL, "study");
-        
+
         sanitizeHTML(study);
 
         // These cannot be set through the API and will be null here, so they are set on update
         Study originalStudy = studyDao.getStudy(study.getIdentifier());
+
         study.setStormpathHref(originalStudy.getStormpathHref());
-        study.setActive(true);
-        // And this cannot be set unless you're an administrator. Regardless of what the 
+        // And this cannot be set unless you're an administrator. Regardless of what the
         // developer set, set these back to the original study.
         if (!isAdminUpdate) {
+            // prevent non-admins update a deactivated study
+            if (!originalStudy.isActive()) {
+                throw new EntityNotFoundException(Study.class, "Study '"+ study.getIdentifier() +"' is deactivated.");
+            }
+
             study.setHealthCodeExportEnabled(originalStudy.isHealthCodeExportEnabled());
             study.setEmailVerificationEnabled(originalStudy.isEmailVerificationEnabled());
         }
+
+        // prevent anyone changing active to false -- it should be done by deactivateStudy() method
+        if (originalStudy.isActive() && !study.isActive()) {
+            throw new IllegalAccessError("Nobody can set field of active to false in update method.");
+        }
```

Comment:
throw BadRequestException, "Study cannot be deleted on update." Otherwise this is returned as a 500 error I believe, and really it's a 400.